04 - CSS JS parser/colorStyles.js

```diff
@@ -1,39 +1,29 @@
-const styles = [{"class":"red","params":[{"name":"color","value":"red"}]},{"class":"green","params":[{"name":"background-color","value":"green"}]},{"class":"blue","params":[{"name":"position","value":"absolute"},{"name":"right","value":"0"},{"name":"top","value":"10px"}]}];
+const styles = [{ "class": "red", "params": [{ "name": "color", "value": "red" }] }, { "class": "green", "params": [{ "name": "background-color", "value": "green" }] }, { "class": "blue", "params": [{ "name": "position", "value": "absolute" }, { "name": "right", "value": "0" }, { "name": "top", "value": "10px" }] }];
 const root = document.body;
 
-
-//root.children.length root.children[i]
-//root.parentElement
-//elem.classList.add/remove/contains
-//elem.style.backgroundColor 
-//$0.style[name] = value;
-
-function addStyle(styleCss, el, colorEl){
-  for (k=0; k<styleCss.length; k++){
-    if (styleCss[k].class === colorEl){
-    for(l=0; l<styleCss[k].params.length; l++){
-    
-    let nameCss = styleCss[k].params[l].name;
-       if(nameCss === "background-color"){
-         nameCss = "backgroundColor";
-       }
-    let valueCss = styleCss[k].params[l].value;
-    
-    el.style[nameCss] = valueCss;
-    }
+function addStyle(styleCss, el, colorEl) {
+  for (k = 0; k < styleCss.length; k++) {
+    if (styleCss[k].class === colorEl) {
+      for (l = 0; l < styleCss[k].params.length; l++) {
+        let nameCss = styleCss[k].params[l].name;
+        if (nameCss === "background-color") {
+          nameCss = "backgroundColor";
+        }
```

Comment:
here you are supporting only one case, but there are many others e.g. 'margin-left', 'word-wrap', 'font-family', 'text-align-last', 'animation-fill-mode' etc. Can you update this code to support all of them?